src/LaunchDarkly.ClientSdk/LdClient.cs

```diff
@@ -463,13 +555,48 @@ public static LdClient Init(Configuration config, Context initialContext, TimeSp
         /// <seealso cref="InitAsync(Configuration, User)"/>
         /// <seealso cref="InitAsync(string, ConfigurationBuilder.AutoEnvAttributes, Sdk.Context)"/>
         /// <seealso cref="Init(Configuration, Sdk.Context, TimeSpan)"/>
+        [Obsolete("Initializing the LDClient without a timeout is no longer permitted to help prevent " +
+                  "consumers from blocking their application execution by mistake when connectivity is poor.  Please " +
+                  "use InitAsync(Configuration, Context, TimeSpan) and specify a max wait time.")]
         public static async Task<LdClient> InitAsync(Configuration config, Context initialContext)
         {
             var c = CreateInstance(config, initialContext, TimeSpan.Zero);
             await c.StartAsync();
             return c;
         }
 
+        /// <summary>
+        /// Creates a new <see cref="LdClient"/> singleton instance and attempts to initialize feature flags
```

Comment:
What is the word "singleton" doing for us here?

Reply:
Helps customer's realize that they don't make multiple of these.

Reply:
Further down is 

        /// You must use one of these static factory methods to instantiate the single instance of LdClient
        /// for the lifetime of your application.